cfg/URG.cfg

```diff
@@ -11,6 +11,7 @@ gen = ParameterGenerator()
 gen.add("tf_prefix",             str_t,    0,                               "tf_prefix for this node's output.",                                                                     "")
 gen.add("frame_id",              str_t,    0,                               "Output frame_id for the laserscan.",                                                                    "laser")
 gen.add("time_offset",           double_t, 0,                               "A manually calibrated offset (in seconds) to add to the timestamp before publication of a message.",    0.0,   -10.0, 10.0)
+gen.add("range_offset",          double_t, 0,                               "Adjustment, in meters, for ranges returned at all angles",                                              0.0,   -10.0, 10.0)
```

Comment:
10m limit is arbitrary, which I have mixed feelings about. _Shouldn't_ be an issue since 10m offset is insane, and setting a limit makes it easier to use the rqt GUI.

Reply:
Wouldn't any value be arbitrary?

Reply:
Yup. But the limits are optional parameters, you can leave them out and dynamic reconfigure won't clamp the parameter at all. The trade-off is the slider in rqt_reconfigure becomes harder to use. There's still some scaling if you try to drag it around, but the slider will be between [-inf, inf].

Reply:
Preference?

Reply:
I guess not setting a limit? It doesn't much matter either way, but if we were doing this as a regular parameter we wouldn't clamp it.